src/idb.filesystem.js

```diff
@@ -34,16 +34,17 @@ if (exports.requestFileSystem || exports.webkitRequestFileSystem) {
 }
 
 // Bomb out if no indexedDB available
-const indexedDB = exports.indexedDB || exports.mozIndexedDB ||
+var indexedDB = exports.indexedDB || exports.mozIndexedDB ||
                   exports.msIndexedDB;
 if (!indexedDB) {
   return;
 }
 
-let IDB_SUPPORTS_BLOB = true;
+var IDB_SUPPORTS_BLOB = false;
```

Comment:
The transpired file that's in dist/ works in all browsers. Put it this way, if this source was written in .ts or .coffee, it would require transpilation to run in older browsers. The same is currently true for es6 features. Ultimately, it doesn't matter how the source is written because we're now providing a built file.